packages/server/graphql/types/JiraIssue.ts

```diff
@@ -71,6 +71,10 @@ const JiraIssue = new GraphQLObjectType<any, GQLContext>({
       type: new GraphQLNonNull(GraphQLString),
       description: 'The plaintext summary of the jira issue'
     },
+    possibleEstimationFieldNames: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLString))),
+      description: 'Field names that exists on the issue and can be used as estimation fields'
+    },
```

Comment:
maybe a comment that this field is computed in the atlassian data loader resolver?